src/pages/components/UserBubble.tsx

```diff
@@ -1 +1,31 @@
-//For that little bubble in the top right, do we still need this?
\ No newline at end of file
+import React, { useState } from 'react';
+import { Dropdown, DropdownTrigger, DropdownMenu, DropdownItem } from '@nextui-org/react';
+import { FaUserCircle } from 'react-icons/fa';
+
+export default function UserBubble() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleMouseEnter = () => {
+    setIsOpen(true);
```

Comment:
These functions are kind of useless, in the future if it's just a one line that isn't really used too much you could make them anonymous functions `() => setIsOpen(true)` but other than that it looks good